src/types/content.rs

```diff
@@ -0,0 +1,176 @@
+use std::collections::HashMap;
+use ::types::messages::{ImageInfo};
```

Comment:
is the `::` prefix in this `use` declaration necessary? if not, please remove (I admit, my own understanding of rust's module system may be at fault here)